Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -87,7 +87,27 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         this._opacity = 0;
         this._opacityPattern = "url(Images/Checkers.svg)";
 
-        this._color = WI.Color.fromString("white");
+        this.color = WI.Color.fromString("white");
+
+        if (colorVariables.length) {
```

Comment:
this should be `colorVariables?.length` since it's not guaranteed that the caller will provide `colorVariables` when created